tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml

```diff
@@ -3286,6 +3286,12 @@
     </match>
     </magic>
   </mime-type>
+  <mime-type type="application/x-amf">
+    <tika:link>https://en.wikipedia.org/wiki/Additive_manufacturing_file_format</tika:link>
+    <root-XML localName="amf"/>
+    <glob pattern="*.scad"/>
```

Comment:
This is the file ending for the OpenSCAD format. File ending for amf is `*.amf`

Reply:
https://en.wikipedia.org/wiki/Additive_manufacturing_file_format Fixed. Thank you.